framework/docs/source/how-to-run-flower-with-deployment-engine.rst

```diff
@@ -20,115 +20,140 @@ Before you start, make sure that:
 
-       * ``flower-supernode``: Name of the SuperNode binary.
-       * | ``--insecure``: This flag tells the SuperNode to operate in an insecure mode, allowing
-         | unencrypted communication.
+       * ``flower-supernode``: Name of the ``SuperNode`` binary.
+       * | ``--insecure``: This flag tells the ``SuperNode`` to operate in an insecure mode, allowing
+         | unencrypted communication. Refer to the :doc:`how-to-enable-tls-connections` guide to learn how to run your ``SuperNode`` with TLS.
        * | ``--superlink 127.0.0.1:9092``: Connect to the SuperLink's Fleet API at the address
-         | ``127.0.0.1:9092``.
-       * | ``--node-config "partition-id=0 num-partitions=2"``: Set the partition ID to ``0`` and the
-         | number of partitions to ``2`` for the SuperNode configuration.
-       * | ``--supernode-address 127.0.0.1:9094``: Set the address and port number where the
-         | SuperNode is listening to communicate with the ClientApp.
-       * | ``--isolation subprocess``: Tells the SuperNode to run the ClientApp in a subprocess.
+         | ``127.0.0.1:9092``. If you had launched the ``SuperLink`` in a different machine, youd replace ``127.0.0.1`` with the public IP of that machine.
+       * | ``--clientappio-api-address 127.0.0.1:9094``: Set the address and port number where the
+         | SuperNode is listening to communicate with the ``ClientApp``.
+       * | ``--node-config "partition-id=0 num-partitions=2"``: The ``ClientApp`` code generated via the ``flwr new`` template expects those two key-value pairs to be defined at run time. Set the partition ID to ``0`` and the number of partitions to ``2`` for the ``SuperNode`` configuration.
 
-2. **Terminal 2** Start the second SuperNode:
+2. **Terminal 2** Start the second ``SuperNode`` after activating your environment:
 
    .. code-block:: shell
 
        $ flower-supernode \
             --insecure \
             --superlink 127.0.0.1:9092 \
-            --node-config "partition-id=1 num-partitions=2" \
-            --supernode-address 127.0.0.1:9095 \
-            --isolation subprocess
-
-Step 4: Start the SuperExec
----------------------------
-
-In a new terminal, start the SuperExec process with the following command:
-
-.. code-block:: bash
+            --clientappio-address 127.0.0.1:9095 \
+            --node-config "partition-id=1 num-partitions=2"
 
-    $ flower-superexec \
-        --insecure \
-        --executor flwr.superexec.deployment:executor \
-        --executor-config 'superlink="127.0.0.1:9091"'
+   .. dropdown:: Understand the command
 
-.. dropdown:: Understand the command
+       * ``--clientappio-api-address 127.0.0.1:9095``: Note that a different port is being used. This is only needed because you are running two ``SuperNodes`` on the same machine. Typically you would run one node per machine and therefore, the ``--clientappio-api-address`` could be omited all together and left with its default value.
```

Comment:
```suggestion
       * ``--clientappio-api-address 127.0.0.1:9095``: Note that a different port is being used. This is only needed because you are running two ``SuperNodes`` on the same machine. Typically you would run one node per machine and therefore, the ``--clientappio-api-address`` could be omitted all together and left with its default value.
```